frontend/src/components/Body/Tarifs/Tariffs.js

```diff
@@ -33,6 +41,12 @@ const StyledButton = styled(Button)`
   }
 ` 
 
+const StyledButtonContainer = styled(Box)`
+  display: flex;
+  justify-content: center;
+  margin-top: auto;
+`;
+
 const StyledBgImg = styled(Box)``;
```

Comment:
Remove this const and add a default "Box" to a component. Leave a corresponding comment like "Box for an image"